atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceRuntimeConfig.java

```diff
@@ -45,6 +45,11 @@ public String type() {
         return TYPE;
     }
 
+    @Value.Default
+    public CassandraServersConfigs.CassandraServersConfig servers() {
+        return ImmutableDefaultConfig.builder().build();
```

Comment:
nit: consider adding `singleton = true` to the `@Value.Immutable` annotation on `DefaultConfig`
```suggestion
        return ImmutableDefaultConfig.of();
```

Reply:
`of` doesn't exist